src/main/java/net/dv8tion/jda/api/components/textinput/TextInput.java

```diff
@@ -30,9 +29,9 @@
 /**
  * Represents a Discord Text input component
  *
- * <p>Must be used in {@link Modal Modals}!
+ * <p>Must be used inside {@link net.dv8tion.jda.api.components.label.Label Labels} only!
  */
-public interface TextInput extends ActionComponent, ActionRowChildComponent
+public interface TextInput extends ActionComponent, LabelChildComponent
```

Comment:
I think `ActionComponent` doesn't even make sense on this, right? You can't disable them and they can't be used outside of modals, so you can't have interactions directly on these either.